README.md

```diff
@@ -13,19 +13,18 @@ examples and guides.
 
 Prometheus, a [Cloud Native Computing Foundation](https://cncf.io/) project, is a systems and service monitoring system. It collects metrics
 from configured targets at given intervals, evaluates rule expressions,
-displays the results, and can trigger alerts if some condition is observed
-to be true.
+displays the results, and can trigger alerts when specified conditions are observed.
 
-Prometheus's main distinguishing features as compared to other monitoring systems are:
+The features that distinguish Prometheus from other metrics and monitoring systems are:
 
-- a **multi-dimensional** data model (time series defined by metric name and set of key/value dimensions)
+- A **multi-dimensional** data model (time series defined by metric name and set of key/value dimensions)
 - PromQL, a **powerful and flexible query language** to leverage this dimensionality
-- no dependency on distributed storage; **single server nodes are autonomous**
-- time series collection happens via a **pull model** over HTTP
-- **pushing time series** is supported via an intermediary gateway
-- targets are discovered via **service discovery** or **static configuration**
-- multiple modes of **graphing and dashboarding support**
-- support for hierarchical and horizontal **federation**
+- No dependency on distributed storage; **single server nodes are autonomous**
+- An HTTP **pull model** for time series collection
+- **Pushing time series** is supported via an intermediary gateway
```

Comment:
We should probably mention that this is only for batch jobs

Reply:
I'm adding in a qualifier as requested.  Git commenting styles vary, and I haven't used the push gateway myself, but I think you're commenting on ` - **Pushing time series** is supported via an intermediary gateway` where I'd only changed capitalization.